src/app.ts

```diff
@@ -410,6 +422,7 @@ function registerAPIRoutes(
   app.get(
     `${basePath}/services/:id`,
     bearerTokenAuth,
+    cache,
     toExpressHandler(servicesController.getService, servicesController)
   );
 
```

Comment:
we should also cache `/services` - @fpersico is it ok to cache `/services`? can you confirm that the app fetches the sender service of a message even if it's not included in the services list?

Reply:
@cloudify `apicache` handle different cache for responses with different headers? I think `/services` returns different responses based on the user that call it.

Reply:
@cloudify The get the list of the messages, extract the list of related servicesids and than load the servicesids not already in cache. The "visible services list" is not used here.

Reply:
@BurnedMarshal it's `sender-services` that is different for each user, `/services` is just a list of services visibile to all users - please add that to the cache

Reply:
fixed in c30fc7f